scalding-commons/src/main/java/com/twitter/scalding/commons/tap/VersionedTap.java

```diff
@@ -93,7 +93,9 @@ public String getSinkPath(JobConf conf) {
   @Override
   public void sourceConfInit(FlowProcess<JobConf> process, JobConf conf) {
     super.sourceConfInit(process, conf);
-    FileInputFormat.setInputPaths(conf, getSourcePath(conf));
+    Path[] inputPaths = FileInputFormat.getInputPaths(conf);
+    inputPaths[inputPaths.length] = new Path(getSourcePath(conf));
```

Comment:
Thanks for the suggestion. 
Because inputPaths[inputPaths.length] is so obviously broken, I wonder if the tests I did on my versioned sources did actually execute this code. I will switch to FileInputFormat#addInputPaths and add a broken test that becomes fixed with this change.